example/src/components/TextInput.tsx

```diff
@@ -69,18 +86,39 @@ export const TextInput = ({
   };
 
   return (
-    <RNTextInput
-      style={GetStyleForTextInput()}
-      placeholder={placeholder}
-      placeholderTextColor={placeholderTextColor}
-      value={value}
-      onFocus={onFocus}
-      onBlur={offFocus}
-      editable={editable}
-      onChangeText={onChangeText}
-      // @ts-ignore
-      colorCursor={TextColors.darkText}
-      selectionColor={TextColors.additionalLightText}
-    />
+    <>
```

Comment:
Wrap it in a `<View`. I suppose it won't happen with the current designs, but if the parent container would have `flexDirection: row` the label would be to right of the text input. We wouldn't like that

Reply:
Done.